.github/workflows/backend-api-spec-update.yml

```diff
@@ -0,0 +1,42 @@
+name: Update backend api spec generated by Kotlin Gradle Plugin
+
+on:
+  pull_request:
+    paths:
+      - 'save-backend/**/*Controller'
+
+jobs:
+  update_backend_api_spec:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Checkout source branch directly, without merging base into it
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Set up Java
+        uses: actions/setup-java@v1
```

Comment:
Why `v1`?

Reply:
Changed to `v3`

Reply:
I believe `v3` requires `distribution` parameter, please compare to other workflows

Reply:
🤦‍♂️ Thanks